e2e/cli-e2e/mocks/fixtures/lcov.info

```diff
@@ -0,0 +1,78 @@
+TN:
```

Comment:
Here I wanted to showcase three types of files of different lengths to ensure that the length of file does not matter, only the total number of entities over all files, meaning:
- The test case needs to include 3 files, one fully covered, one partly covered, one with no coverage.
- As explained with the single file record, each file should have multiple entities of each type spread in a realistic scenario.
- The lengths need to differ so I went between 5 and 10, very short file examples.
- In order to make it easier on the debugger, I included a comment explaining how the data are parsed into results right above the test.

In my opinion, mashing all entities in 3 lines would not be realistic and it would make e.g. testing line ranges impossible on such a small scale. This test case utilises as many use cases as possible in a limited space. Again, open to counterarguments.

Reply:
I always think about our eslint diffs and how I debug it when i see bigger fixtures. :D 
The less the better but it's definitely no change request from me.

Reply:
The snapshots do not exceed 200 lines so we should be fine (ESLint snapshots are 1000+ lines long).